src/lib/registry/severity-registry.js

```diff
@@ -0,0 +1,47 @@
+const Severity = require('../severity');
+const EmptyRegistry = require('./empty-registry');
+const { throwIf } = require('../utils/type-checks');
+
+class SeverityRegistry extends EmptyRegistry {
+    constructor() {
+        super();
+    }
+
+    add(name, instance) {
+        throwIf.not.string(name, 'name');
+        throwIf.not.instanceof(instance, Severity, 'instance', 'Severity');
+
+        this._registry[name] = instance;
+    }
+
+    register(level, name) {
+        throwIf.not.string(name, 'name');
+        throwIf.not.number(level, 'level');
+
+        const severity = new Severity(level, name);
+
+        const nameUpper = name.toUpperCase();
+        const nameLower = name.toLowerCase();
+
+        this._registry[nameUpper] = severity;
+        this._registry[nameLower] = severity;
+
+        return this;
+    }
+
+    get(query) {
+        if (query instanceof Severity) return query;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/loggin-js/loggin-js/pullRequest?prid=7629800)
```suggestion
        if (query instanceof Severity) {return query;}
```